src/components/accordion/accordion.e2e.ts

```diff
@@ -46,14 +46,14 @@ describe('sbb-accordion', () => {
   it('should set accordion context on expansion panel when removing and adding expansion-panels', async () => {
     let panels: SbbExpansionPanelElement[];
 
-    element.querySelector('sbb-expansion-panel').remove();
+    element.querySelector('sbb-expansion-panel')?.remove();
```

Comment:
In test files, should we not generally use "!" instead of "?"? We know what we are doing.

Reply:
Good point, I'll re-check tests 👍